pkg/operator/operator/k8s_specs.go

```diff
@@ -107,11 +107,18 @@ func tfAPISpec(api *spec.API, prevDeployment *kapps.Deployment) *kapps.Deploymen
 	}
 
 	servingImage := config.Cluster.ImageTFServe
+	apiImage := config.Cluster.ImageTFAPI
```

Comment:
I agree with keeping it simple. And since you no longer have to update the cluster if the user wants a different base image, then the incentive to use `cluster.yaml` to update the images is less significant.